### ISSUE TITLE
Editing beatmap description on new website replaces  "." and ":" in the forum stub information section with stuff

### ISSUE THREAD

**Issue by dorothy3242** [ppy/osu-web#1860]
Sometimes " ' " too but that doesn't seem to always happen

This breaks all of the stub's information section links so it seems worth mentioning even if forums and their stubs for beatmaps are on the way out, because this means the mapset's actual page is inaccessible from said stub unless the mapper updates the set or manually fixes the links. 🤷

ex:
`[b]Information:[/b] [url=https://osu.ppy.sh/s/688590]Scores/Beatmap Listing[/url]` would become
`[b]Information:[/b] [url=https&#58;//osu&#46;ppy&#46;sh/s/688590]Scores/Beatmap Listing[/url]`

which then leads to this when clicked, instead of the map's information page:
![image](https://user-images.githubusercontent.com/33286824/33434083-2781343a-d5ac-11e7-84bc-2439810f4094.png)

**Comment by notbakaneko** [comment]
can you provide some links to the beatmaps/threads where this happens?

**Comment by dorothy3242** [comment]
I actually accidentally did this to all of my ranked maps before realizing it was happening :s (wasn't this supposed to be disabled for ranked maps? I edited my ranked maps to see if it would work.). Then did my best to manually fix it. I ended up replacing most difficulty links with the direct beatmap id .osu links from the information page however because I couldn't always figure out how to fix the links with the difficulty name in them...

I'll re-do it for example's sake.

If you do it multiple times, it gets worse and worse until you update the map, because the & symbol in the stuff that replaces "." and ":" also turns into stuff. Then the one at the front of this turns into stuff, etc.

https://osu.ppy.sh/forum/p/5568332 < after one edit
https://osu.ppy.sh/forum/p/6329835 <example of the horror that happens after multiple edits.
